src/components/summary/_macro.njk

```diff
@@ -9,24 +9,27 @@
     {% if params.hub %}
         {% set className = className + " ons-summary--hub" %}
     {% endif %}
+
+
     <div class="{{ className }}">
         {% for summary in params.summaries %}
             {% if summary.summaryTitle %}
                 <h2 class="ons-summary__title ons-u-mb-m{{ " ons-u-mt-m" if loop.index > 1 else "" }}">{{ summary.summaryTitle }}</h2>
                 {% set titleSize = "3" %}
             {% endif %}
             {% for group in summary.groups %}
-                <div {% if group.id %}id="{{ group.id }}" {% endif %} class="ons-summary__group">
+                <div {% if group.id %}id="{{ group.id }}" {% endif %} class="{{"ons-summary__group--card" if params.variant == 'card' and not params.hub else "ons-summary__group"}}">
                     {% if group.groupTitle %}
-                        <h{{ titleSize }} class="ons-summary__group-title">{{ group.groupTitle }}</h{{ titleSize }}>
+                        <h{{ titleSize }} class="{{"ons-summary__group-title--card" if params.variant == 'card' and not params.hub else "ons-summary__group-title"}}">{{ group.groupTitle }}</h{{ titleSize }}>
                     {% endif %}
                     {% if group.rows %}
-                        <div class="ons-summary__items">
+                        <div class="{{"ons-summary__items--card" if params.variant == 'card' and not params.hub else "ons-summary__items"}}">
 
                             {% for row in group.rows %}
                                 {% set itemClass = "" %}
                                 {% if row.error %} {% set itemClass = " ons-summary__item--error" %}{% endif %}
                                 {% if row.total %} {% set itemClass = itemClass + " ons-summary__item--total" %}{% endif %}
+                                {%if params.variant == "card" %} {% set itemClass = "--card" %} {% endif %}
```

Comment:
This logic wont work if `row.error` or `row.total` are set alongside `params.variant` being set to "card"